package.json

```
@@ -57,7 +57,7 @@
     "rxjs": "6.x"
   },
   "devDependencies": {
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "^5.8.2-pre.2301",
```

Always reset to `next`.

```suggestionj
    "@openmrs/esm-framework": "next",
```